server/multitenancy/tenant_index.ts

```diff
@@ -41,6 +41,7 @@ export async function setupIndexTemplate(
     await esClient.indices.putTemplate({
       name: 'tenant_template',
       body: {
+        order: 2147483647,
```

Comment:
Can you add a short inline comment as to why this value should be high?

Reply:
Note; If you create a const for the order value as DC recommended with a meaningful name that would also be enough to protect the 'why' of this value.